src/amo/components/Page/index.js

```diff
@@ -73,9 +76,14 @@ export const PageBase = ({
     }
   }
 
+  const pageAddonType = addonType || location.query.addonType;
```

Comment:
This does not look right to be honest. I don't think we should be using the query string to carry information that we already have stored somewhere else, which I believe is the case given that this query parameter is set on _internal_ URLs.

It's probably fine to inject `addonType` in `Page` but the value should be used as is. I'll have another comment about how we could potentially achieve that in the `Addon` component.